src/options.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8"/>
+  </head>
+
+  <body>
+    <form>
+      <input type="checkbox" id="bookmarksPermissions"/>
+      <label>Enable Bookmark Permissions</label>
```

Comment:
Let's call this "Enable Bookmark Menus"? Since "permissions" is just the technical thing we need, and "menus" is the user-facing feature.